coralproject/talk
# Clicking '# comments' tab refresh all the comments back

## What does this PR do?

It resolves story https://www.pivotaltracker.com/story/show/143520139

It adds an action to the tab "# Comments". Now it is clickable and bring back all the comments.

## How do I test this PR?

- Go to a permalink
- Click on the tab "# comments"
- Check that you get all the comments back
